integration-tests/actions/vrfv2_actions/vrfv2_steps.go

```diff
@@ -308,6 +349,80 @@ func SetupVRFV2Environment(
 	return vrfv2Contracts, subIDs, &data, nil
 }
 
+func SetupVRFV2WrapperEnvironment(
+	env *test_env.CLClusterTestEnv,
+	vrfv2Config vrfv2_config.VRFV2Config,
+	linkToken contracts.LinkToken,
+	mockNativeLINKFeed contracts.MockETHLINKFeed,
+	coordinator contracts.VRFCoordinatorV2,
+	keyHash [32]byte,
+	wrapperConsumerContractsAmount int,
+) (*VRFV2WrapperContracts, *uint64, error) {
+	// Deploy VRF v2 direct funding contracts
+	wrapperContracts, err := DeployVRFV2DirectFundingContracts(
+		env.ContractDeployer,
+		env.EVMClient,
+		linkToken.Address(),
+		mockNativeLINKFeed.Address(),
+		coordinator,
+		wrapperConsumerContractsAmount,
+	)
+	if err != nil {
+		return nil, nil, err
+	}
+	err = env.EVMClient.WaitForEvents()
```

Comment:
Is there a way to do assignment expressions in Go (assign the variable and check the condition in one line)? Or this is the standard way to do error handling in Go? Just curious if anyone knows... I have very little knowledge of Go.

Reply:
I guess it would look smth like this:

```
	if err := env.EVMClient.WaitForEvents(); err != nil {
		return nil, nil, fmt.Errorf("%s, err %w", ErrWaitTXsComplete, err)
	}
```
but its matter of preference